### ISSUE TITLE
Support right click context menu for text inputs in the Edit Entry and Message Text dialogs

### ISSUE THREAD

**Issue by ChrisSchaller** [Microsoft/AzureStorageExplorer#8881]
### Problem

I am unable to efficiently operate the Add Message or Edit Entry dialog boxs using the mouse alone. This is a significant accessibility issue for people with physical impairments who struggle to use keyboard shortcuts or to switch between mouse and keyboard. Storage Explorer lack keyboard accessibility options in most of it's interfaces, it is a very mouse focused interface, however when it comes to text entry, standard context menus that can be driven from the mouse are not supported.

### Desired Solution

Right click on the Message Text input (or pressing the menu key, or shift+F10) should display the standard context menu for text entry that includes options like 'Paste'

### Alternatives and Workarounds

The only workaround is to click or tab into the Message Text and then to use the keyboard short cut ctrl+v to paste content into the messages box or to manually type the message.

### Additional Context

Messages for queues are often very specific in their content or involve unique Ids that are not reliable to enter manually by hand, this means that pasting exact content from other sources is most commonly the primary function that data explorer operators are likely to require. It is strange that this OS standard feature is not present in this application.

**Comment by craxal** [comment]
According to Electron documentation, context menus do not appear by default. They need to be manually configured. Also according to Electron, there is a way to open generalized context menus for input elements. We have multiple "windows" in the app, so we'd have to add code in several places.

https://www.electronjs.org/docs/latest/tutorial/context-menu
